topics/releases.md

```diff
@@ -1,144 +1,98 @@
+description: How new versions of Valkey released and supported
 aliases:
     - /topics/releases
 ---
 
-Valkey is system software and a type of system software that holds user data, so
-it is among the most critical pieces of a software stack.
+Valkey is usually among the most critical pieces of a software stack.
+For this reason, Valkey's release cycle prioritizes highly stable releases at the cost of slower release cycles.
 
-For this reason, Valkey's release cycle is such that it ensures highly stable
-releases, even at the cost of slower cycles.
-
-New releases are published in the [Valkey GitHub repository](https://github.com/valkey-io/valkey/releases).
-
-## Release cycle
-
-A given version of Valkey can be at three different levels of stability:
-
-* Unstable
-* Release Candidate
-* Stable
-
-### Unstable tree
-
-The unstable version of Valkey is located in the `unstable` branch in the
-[Valkey GitHub repository](https://github.com/valkey-io/valkey).
-
-This branch is the source tree where most of the new features are under
-development. `unstable` is not considered production-ready: it may contain
-critical bugs, incomplete features, and is potentially unstable.
-
-However, we try hard to make sure that even the unstable branch is usable most
-of the time in a development environment without significant issues.
-
-### Release candidate
-
-New minor and major versions of Valkey begin by branching off the `unstable`
-branch. The branch name is the target release on the form *major.minor*.
-Subsequent patch releases are made on the same branch.
-
-For example, when Valkey 7.2.5 was released, the release was made on the `7.2`
-branch, which had been branched off from `unstable` earlier.
-
-Bug fixes and new features that can be stabilized during the release's time
-frame are committed to the unstable branch and backported to the release
-candidate branch. The `unstable` branch may include additional work that is not
-a part of the release candidate and scheduled for future releases.
-
-The first release candidate, or RC1, is released once it can be used for
-development purposes and for testing the new version. At this stage, most of
-the new features and changes the new version brings are ready for review, and
-the release's purpose is collecting the public's feedback.
-
-Subsequent release candidates are released every three weeks or so, primarily
-for fixing bugs. These may also add new features and introduce changes, but at
-a decreasing rate and decreasing potential risk towards the final release
-candidate.
-
-### Stable tree
-
-Once development has ended and the frequency of critical bug reports for the
-release candidate wanes, it is ready for the final release. At this point, the
-release is marked as stable and is released with "0" as its patch-level
-version.
+All Valkey releases are published in the [Valkey GitHub repository](https://github.com/valkey-io/valkey/releases).
 
 ## Versioning
```

Comment:
We should be more clear about the backwards compatibility about:
1. RDB Versioning
2. API versioning
3. Replication protocol
4. Cluster protocol.

Reply:
Modules?